<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Меню</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_team_stats">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>231</width>
       <height>88</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #cf617a; /* Розовый цвет фона */
    color: white; /* Белый цвет текста */
    border: none; /* Убрать рамку */
    border-radius: 20px; /* Закругление углов для овала */
     padding: 10px 20px; /* Внутренние отступы */
    font-size: 16px; /* Размер шрифта */
    /* Задаем овальную форму */
}

QPushButton:hover {
    background-color: #c36077; /* Цвет фона при наведении */
}

QPushButton:pressed {
    background-color: #703845; /* Цвет фона при нажатии */
}</string>
     </property>
     <property name="text">
      <string>Выгрзука таблиц</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stats">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>310</y>
       <width>231</width>
       <height>88</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #cf617a; /* Розовый цвет фона */
    color: white; /* Белый цвет текста */
    border: none; /* Убрать рамку */
    border-radius: 20px; /* Закругление углов для овала */
     padding: 10px 20px; /* Внутренние отступы */
    font-size: 16px; /* Размер шрифта */
    /* Задаем овальную форму */
}

QPushButton:hover {
    background-color: #c36077; /* Цвет фона при наведении */
}

QPushButton:pressed {
    background-color: #703845; /* Цвет фона при нажатии */
}</string>
     </property>
     <property name="text">
      <string>Статистика</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-20</y>
       <width>811</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/обои.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/Telegram Desktop/yfv yv.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>pushButton_team_stats</zorder>
    <zorder>pushButton_stats</zorder>
    <zorder>label_9</zorder>
   </widget>
   <widget class="QWidget" name="">
    <attribute name="title">
     <string>Личный кабинет</string>
    </attribute>
    <widget class="QLabel" name="label_user_info">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>551</width>
       <height>201</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>-20</y>
       <width>831</width>
       <height>561</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/обои.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/1734596351450.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>290</y>
       <width>141</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/крем.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Личный кабинет</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>440</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/пуш.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>240</y>
       <width>141</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/помада.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>340</y>
       <width>161</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../Downloads/палетка.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_user_info</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
